File: src/main/java/com/kevin/guess/Game.java
package com.kevin.guess;

import java.util.Random;
import java.util.Scanner;

public class Game {
    public static void main(String[] args) {
        Random random = new Random();
        int secrect = random.nextInt(10)+1;
        System.out.println(secrect);

        int number = 0;
        while (number != secrect){
            Scanner scanner = new Scanner(System.in);
            System.out.print("Please enter your number : ");
            number = scanner.nextInt();
            if (number < secrect){
                System.out.println("Higher!!");
            }else if(number > secrect){
                System.out.println("Lower!!");
            }else{
                System.out.println("Great, the answer is : " + number);
            }

        }
//        int i = 0;
//       do {
//           System.out.println(i);
//           i++;
//       }while (i <= 10);
    }
}

File: src/main/kotlin/com/kevin/guess/Game.kt
package com.kevin.guess

import java.util.*

fun main() {
//    val scanner = Scanner(System.`in`)
    var number = 0
    val secrect = Random().nextInt(10)+1
    println(secrect)

    while(number != secrect){
        print("Please enter a number : ")
        number = readLine()!!.toInt()
//        number = scanner.nextInt()
        if (number > secrect){
            println("Lower!!")
        }else if (number < secrect){
            println("Highter!!")
        }else{
            println("好極了! the answer is : $secrect")
        }
    }
}